File: AtCoder_Beginners_Selection/atcorder_card_game_for_two.js
const { constants } = require("buffer");

function Main(input) {
  // ファイル読み込み
  const text = input.split("\n");
  const a = parseInt(text[0], 10),
        b = text[1].split(" ").map(x => +x).slice().sort((a, b) => b - a);

  let asum = 0,
      bsum = 0;
  
  // console.log(a, b);

  // a:0,2,4
  // b:1,3,5

  for (let i = 0; i < a; i++) {
    (i % 2 == 0) ? asum += b[i] : bsum += b[i];
  }
  
  const result = (asum <= bsum) ? bsum - asum : asum - bsum;

  console.log(result);
}

Main(require("fs").readFileSync("/dev/stdin", "utf8"));
// Main(require("fs").readFileSync("sample.txt", "utf8"));

function Main(input) {
	
}

Main(require("fs").readFileSync("sample.txt", "utf8"));  // テスト用
// Main(require("fs").readFileSync("/dev/stdin", "utf8"));   // 提出用

File: AtCoder_Beginners_Selection/atcorder_card_game_for_two.py
N = int(input())
a_list = list(map(int, input().split(" ")))
a_list = sorted(a_list, reverse=True)

a_sum = 0
b_sum = 0

for i in range(N):
  if i == 0 or i % 2 == 0:
    a_sum += a_list[i]
  else:
    b_sum += a_list[i]

print(a_sum - b_sum)
